vision_agent/tools/tools.py

```diff
@@ -2224,6 +2354,44 @@ def _plot_counting(
     return image
 
 
+def convert_florence2_sam2_to_flux_mask(
```

Comment:
I would suggest just doing this internally in the flux_image_painting function so the LLM has less function calls to worry about.

Reply:
and we also have this interesting result of object detection, where it deals with the masking without the util function:


```python
import os
import numpy as np
from vision_agent.tools import *
from typing import *
from pillow_heif import register_heif_opener
register_heif_opener()
import vision_agent as va
from vision_agent.tools import register_tool


from vision_agent.tools import *
import numpy as np

def change_hair_color(image_path):
    # Load the image
    image = load_image(image_path)
    
    # Use florence2_sam2_image to segment the hair
    hair_segments = florence2_sam2_image("hair", image)
    
    # Convert the segmentation mask
    height, width = image.shape[:2]
    mask = np.zeros((height, width), dtype=np.uint8)
    for segment in hair_segments:
        mask_array = segment['mask']
        mask[mask_array == 1] = 255  # Set hair pixels to white (255)
    
    # Use flux_image_inpainting to change the hair color to green
    inpainted_image = flux_image_inpainting(
        "green hair",
        image,
        mask,
    )[0]
    
    # Save the resulting image
    output_path = "green_hair_result.png"
    save_image(inpainted_image, output_path)
    
    return output_path

# The function can be called like this:
result_path = change_hair_color("./data/image.png")
``